<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="toolTip">
   <string>学生管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 104, 16);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 145, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 125, 11);
</string>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="bg_title" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 104, 16);
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>253</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lb_username">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>用户名：admin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_exit">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.500318, y1:0.505909, x2:0.506, y2:1, stop:0 rgba(221, 23, 5, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 214, 117);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0" colspan="14">
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>姓名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>年龄</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>年级</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>班级</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>学号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>手机号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>微信</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>千奇百怪</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item row="0" column="5">
           <property name="text">
            <string>123456</string>
           </property>
          </item>
          <item row="0" column="6">
           <property name="text">
            <string>12345678900</string>
           </property>
          </item>
          <item row="0" column="7">
           <property name="text">
            <string>wechat</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_simulation">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>模拟数据</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_clear">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
        <item row="2" column="12">
         <widget class="QLabel" name="lb_cnt">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLineEdit" name="le_search">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="11">
         <widget class="QPushButton" name="btn_search">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_del">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_update">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_add">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>增加</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_simulation2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>模拟数据</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="12">
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>新建行</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>用户名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>密码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>权限</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>qqbg</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>123456</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>admin</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLineEdit" name="le_search2">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QPushButton" name="btn_search2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QLabel" name="lb_cnt2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_del2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_update2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_add2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 184, 112);</string>
          </property>
          <property name="text">
           <string>增加</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
